torch/csrc/autograd/python_legacy_variable.cpp

```diff
@@ -48,9 +48,9 @@ static PyObject *THPVariable_pynew(PyTypeObject* type, PyObject *args, PyObject
     // by nn.Parameter() with no arguments.
     auto scalar_type = torch::tensors::get_default_scalar_type();
     auto var = at::empty({0}, torch::tensors::get_default_tensor_type().options(scalar_type));
-    tensor = static_cast<Variable&>(var).data();
+    tensor = static_cast<Variable&>(var);
   } else if (THPVariable_Check(data)) {
-    tensor = ((THPVariable*)data)->cdata.data();
+    tensor = ((THPVariable*)data)->cdata;
```

Comment:
It's okay to remove the `.data()` call in these two statements, because we only call `make_variable(...)` on `tensor` in the rest of the function, which ignores `tensor`'s original AutogradMeta and creates a new AutogradMeta.